flotiq/flotiq-codegen-ts
# #24627 codegen watch param

**Instalation:**

`npm install git+https://github.com/flotiq/flotiq-codegen-ts.git#feature/24627-codegen-watch-param`

**Testing:**
 - run `npx flotiq-codegen-ts --watch --compiled-js`
 - pass your flotiq api key
 - make change in content on your flotiq account
 - in worst case you will have to wait 10 seconds untill codegen detect the change, then you should see informtion about genereting new sdk
 - changes in content on your flotiq account shoud be visible in SDK (**you may have to select option Reload  from disk to see the changes**)
